usaspending_api/api_docs/markdown/endpoints.md

```diff
@@ -25,43 +25,43 @@ Endpoints do not currently require any authorization.
 ### Status Codes <a name="status-codes"></a>
 In general, status codes returned are as follows:
 
-* 200 if successful
+* 200 if the request is successful
 * 400 if the request is malformed
```

Comment:
could add
422 if the request is technically valid but doesn't conform with all constraints

Reply:
Added to the keyword search api endpoints!